addkernels/addkernels.cpp

```diff
@@ -104,7 +104,7 @@ void PrintHelp()
     std::cout << "           -n[o-recurse] : dont expand include files recursively. Default: off"
               << std::endl;
     std::cout << "           -m[ark-includes] : mark variables that represent include files with "
-                 "'__INC'. Default: off"
+                 "'_INC'. Default: off"
```

Comment:
[Recommendation] This is fine, but maybe this:
```suggestion
                 "'_INCLUDE'. Default: off"
```
would be cleaner? Up to you.

Reply:
Changed as you suggested